hashing/src/lib.rs

```diff
@@ -20,12 +20,25 @@ use blake2::{
     VarBlake2b,
 };
 use datasize::DataSize;
-use hex_buffer_serde::{Hex, HexForm};
 use itertools::Itertools;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-use casper_types::bytesrepr::{self, FromBytes, ToBytes};
+use casper_types::{
+    bytesrepr::{self, FromBytes, ToBytes},
+    checksummed_hex::{self, ChecksummedHex, ChecksummedHexForm},
+};
+
+/// Possible hashing errors.
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Incorrect digest length {0}.")]
+    /// The digest lenghth was an incorrect size.
+    IncorrectDigestLength(usize),
```

Comment:
Can we include information on the expected length?

Reply:
Yep, done